src/pages/index.tsx

```diff
@@ -1,8 +1,7 @@
 import React from 'react';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import GradeBook from '../GradeBook';
-import { SignIn, SignUp } from './accounts';
-import ConfirmSignUp from './accounts/ConfirmSignUp';
+import { SignIn, SignUp, ConfirmSignUp, ForgotPassword } from './accounts';
```

Comment:
@BraidenCutforth Does this work for what you were suggesting?

Reply:
Exactly!